pants.toml

```diff
@@ -130,7 +130,7 @@ args = [ "--suppress-no-test-exit-code" ]
 
 [mypy]
 lockfile = "3rdparty/python/mypy.lockfile"
-version = "mypy==0.812"
+version = "mypy==0.910"
```

Comment:
This is [what Pants uses now](https://www.pantsbuild.org/docs/reference-mypy#advanced-options) by default, so we could arguably remove this line.

By removing this, I think we could get away with [using the lockfile that Pants generates](https://www.pantsbuild.org/docs/reference-mypy#section-lockfile), meaning we could get rid of ours.